functions/updateSprintStatusAPI.js

```diff
@@ -17,7 +17,12 @@ exports.updateSprintStatus = functions.https.onRequest((request, response) => {
     cors(request, response, () => {
         const sprintStatus = request.body.data.SprintStatus;
         const currentSprintName = request.body.data.CurrentSprintName;
+        const appKey = request.body.data.AppKey;
         console.log(currentSprintName);
+        db.collection("Organizations").where("AppKey", "==", appKey).get().then((org) => {
+            org.forEach((doc) => {
+                documentID = doc.data().OrganizationDomain;
+            });
         db.collection("Main").doc(currentSprintName).update({
```

Comment:
Add correct path to update status. It should be inside the organization
Organization-domain-Sprints-currentsprintName